src/instruments/src/Common/simVars.tsx

```diff
@@ -289,11 +286,32 @@ const SimVarProvider: React.FC = ({ children }) => {
 export const useSimVar = (
     name: string,
     unit: UnitName,
-    maxStaleness = 0,
+    refreshRate = 0,
 ): [SimVarValue, (newValueOrSetter: SimVarValue | SimVarSetter
 ) => void] => {
-    const value = useSimVarValue(name, unit, maxStaleness);
-    const setter = useSimVarSetter(name, unit);
+    const lastUpdate = useRef(Date.now() - refreshRate - 1);
+
+    const [value, setValue] = useState(() => SimVar.GetSimVarValue(name, unit));
+
+    const updateCallback = useCallback(() => {
+        const newValue = SimVar.GetSimVarValue(name, unit);
```

Comment:
Can’t this go inside the refreshRate check?

Reply:
fixed